flake.nix

```diff
@@ -12,46 +15,33 @@
     nix-helm.inputs.nixpkgs.follows = "nixpkgs";
 
     std = {
-      url = "github:divnix/std";
+      url = "github:divnix/std/v0.24.0-1";
       inputs.nixpkgs.follows = "nixpkgs";
       inputs.n2c.follows = "n2c";
       inputs.devshell.follows = "devshell";
     };
   };
 
-  outputs = {
-    std,
-    nix-helm,
-    ...
-  } @ inputs:
-    inputs.flake-parts.lib.mkFlake {inherit inputs;} {
-      imports = with inputs; [
-        std.flakeModule
-        devshell.flakeModule
-      ];
-      systems = ["x86_64-linux"];
-
-      perSystem = {
-        pkgs,
-        system,
-        ...
-      }: {
-        legacyPackages.cardano-services = import ./nix/cardano-services/deployments {inherit pkgs nix-helm inputs;};
-      };
-
-      std.grow = {
-        cellsFrom = ./nix;
-        cellBlocks = with std.blockTypes; [
-          # Software Delivery Lifecycle (Local Development Environment)
-          (devshells "envs")
-          (runnables "jobs")
-          # Software Delivery Lifecycle (Packaging Layers)
-          # For deeper context, please consult:
-          #   https://std.divnix.com/patterns/four-packaging-layers.html
-          (installables "packages" {ci.build = true;})
-          (runnables "operables")
-          (containers "oci-images" {ci.publish = true;})
-        ];
-      };
-    };
+  # --- Flake Local Nix Configuration ----------------------------
+  nixConfig = {
+    # still used by single-user-mode (e.g. ci)
+    extra-substituters = [
+      "https://cache.iog.io"
+      "s3://lace-nix-cache?region=us-east-1"
+    ];
+    extra-trusted-substituters = [
+      "https://cache.iog.io"
+      "s3://lace-nix-cache?region=us-east-1"
+    ];
+    extra-trusted-public-keys = [
+      "hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ="
+      "nixbuild.net/lace@iohk.io-1:sEHlRBG/EcTkef5vJx2LmDPxpe8kln81TQuZyb9TdJY="
+      "lace:doMHHeFIW0/T/Gw5y+S6OfhRVC5Imhm28rptgdLRBn4="
+    ];
+    allow-import-from-derivation = "true";
+  };
+  # --------------------------------------------------------------
+
+
+  outputs = inputs: import ./outputs.nix inputs;
```

Comment:
can we keep std stuff here, and import perSystem block with devshell somewhere from `nix` directory?